test/evergreen/code_coverage/parallel_code_coverage.py

```diff
@@ -74,7 +80,8 @@ def main():
     parser.add_argument('-s', '--setup', action="store_true",
                         help='Perform setup actions from the config in each build directory')
     parser.add_argument('-v', '--verbose', action="store_true", help='Be verbose')
-    parser.add_argument('-p', '--python', action="store_true", help='Base name for the build directories')
+    parser.add_argument('-p', '--python', action="store_true", help='Run on only python tests in code coverage')
+    parser.add_argument('-o', '--other', action="store_true", help='Run any other tests that are not python in code coverage')
```

Comment:
We always want one of these arguments enabled. We could unify into a single `--bucket python` style argument

Reply:
fixed [23a5fd7](https://github.com/wiredtiger/wiredtiger/pull/10877/commits/23a5fd7a53ea96913509cc005a6ac5d16a1006d1)